data/formatters/macos.yaml

```diff
@@ -349,6 +349,53 @@ short_source: 'PLIST'
 source: 'MacOS launchd plist'
 ---
 type: 'conditional'
+data_type: 'macos:loginitems:entry'
+message:
+- 'Login item: {name}'
+- 'Points to: {target_path}'
+- 'CNID path: {cnid_path}'
```

Comment:
@Spferical are you using this value in any specific way?

It current form is unconventional, given that this "path" starts with the parent not root

Reply:
Not specifically. I was interested in slurping up any easy-to-collect values related to bookmark rename detection if it could be relevant for forensics, but it's not a priority.